gateway/gateway_test.go

```diff
@@ -2747,3 +2751,127 @@ func waitForBackendConfigInit(gw *Handle) {
 		2*time.Second,
 	).Should(BeTrue())
 }
+
+func TestLeakyUploader(t *testing.T) {
+	// Setup Minio test container
+	pool, err := dockertest.NewPool("")
+	require.NoError(t, err)
```

Comment:
If after the require you do `pool.MaxWait = time.Minute` then you could add `t.Parallel()` to the 2 tests below.